pkg/apis/configuration/v1alpha1/types.go

```diff
@@ -215,6 +215,9 @@ type MetricSpec struct {
 	HTTP *MetricHTTP `json:"http,omitempty"`
 	// +optional
 	Rules []MetricsRule `json:"rules,omitempty"`
+	// Latency distribution buckets. If not set, the default buckets are used.
+	// +optional
+	LatencyDistributionBuckets []int `json:"latencyDistributionBuckets,omitempty"`
```

Comment:
q: is there a semantic meaning between "unset" and "empty set", i.e. should this be a pointer to slice to differentiate `nil` pointer and empty slice?
I ask because it has been labeled as optional.

Reply:
should we call this `CustomLatencyDistributionBuckets` to differentiate from default?

Reply:
Tbh, I actually prefer LatencyDistributionBuckets. Although the user doesn't necessarily see it, the field itself (at least internally) gets set to the default value when unset, rather than acting as some kind of override. A good visualisation would be the API docs reference table whereby one of the columns is the default. If it was an override kind of field then the default would be nil rather than the actual end result value.
A lot of Dapr Configuration APIs would end up with Custom prefixes under the logic they customise a behaviour imo, which I don't think we want.